frontend/demo/component/avatar/avatar-menu-bar.ts

```diff
@@ -0,0 +1,56 @@
+import '../../init'; // hidden-full-source-line
+
+import { html, LitElement, customElement, internalProperty } from 'lit-element';
+import '@vaadin/vaadin-avatar/vaadin-avatar';
+import '@vaadin/vaadin-menu-bar/vaadin-menu-bar';
+import { applyTheme } from 'themes/theme-generated.js';
+import { getPeople } from '../../domain/DataService';
+import { MenuBarItem } from '@vaadin/vaadin-menu-bar/vaadin-menu-bar';
+import { AvatarElement } from '@vaadin/vaadin-avatar/vaadin-avatar';
+
+@customElement('avatar-basic')
```

Comment:
Conflicting custom element name
```suggestion
@customElement('avatar-menu-bar')
```

Reply:
Done.